src/modules/kafka/kafka-container.ts

```diff
@@ -6,27 +6,48 @@ import { StartedTestContainer } from "../..";
 import { AbstractStartedContainer } from "../abstract-started-container";
 import { Host } from "../../docker/types";
 import { InspectResult } from "../../docker/functions/container/inspect-container";
-import { dockerClient } from "../../docker/docker-client";
 
 const KAFKA_PORT = 9093;
 const KAFKA_BROKER_PORT = 9092;
 const DEFAULT_ZOOKEEPER_PORT = 2181;
 
 export const KAFKA_IMAGE = "confluentinc/cp-kafka:5.5.4";
 
+export interface SaslSslListenerOptions {
```

Comment:
Currently only the containers are exported from the Kafka module: https://github.com/testcontainers/testcontainers-node/blob/51654f5cceb7c8e2757d82102761d345fbe24979/src/index.ts#L17. I think it's OK not to export these SASL interfaces but maybe users would want them? What do you think?

Reply:
I think it's a good idea. The intention is to provide the config inline. Done.